src/si/electric_charge.rs

```diff
@@ -98,6 +102,8 @@ mod tests {
             test::<i::abampere, t::second, q::abcoulomb>();
             test::<i::statampere, t::second, q::statcoulomb>();
 
+            test::<i::atomic_unit_of_charge_per_second, t::second, q::atomic_unit_of_charge>();
+
```

Comment:
Github won't let me use the full context for the changes, so I had to paste a diff below.

Put the elementary charge and atomic unit of charge tests in the same order as the unit definitions. For elementary charge possibly use elementary charge per second (see electric current review) or fallback to atomic unit of charge per second.

```diff
diff --git a/src/si/electric_charge.rs b/src/si/electric_charge.rs
index c749ec6..ea7406b 100644
--- a/src/si/electric_charge.rs
+++ b/src/si/electric_charge.rs
@@ -98,12 +98,12 @@ mod tests {
             test::<i::zeptoampere, t::second, q::zeptocoulomb>();
             test::<i::yoctoampere, t::second, q::yoctocoulomb>();
 
+            test::<i::elementary_charge_per_second, t::second, q::elementary_charge>();
+            test::<i::atomic_unit_of_charge_per_second, t::second, q::atomic_unit_of_charge>();
             test::<i::ampere, t::hour, q::ampere_hour>();
             test::<i::abampere, t::second, q::abcoulomb>();
             test::<i::statampere, t::second, q::statcoulomb>();
 
-            test::<i::atomic_unit_of_charge_per_second, t::second, q::atomic_unit_of_charge>();
-
             fn test<I: i::Conversion<V>, T: t::Conversion<V>, Q: q::Conversion<V>>() {
                 Test::assert_approx_eq(&ElectricCharge::new::<Q>(V::one()),
                     &(ElectricCurrent::new::<I>(V::one()) * Time::new::<T>(V::one())));
```